<?xml version="1.0" ?>
<launch>

  <arg name="camera_name"	default="b_bot_outside_camera"/>


  <!-- Launch motion_detector -->
  <include file="$(dirname)/run.launch">
    <arg name="camera_info_topic" value="/camera_multiplexer/camera_info"/>
    <arg name="image_topic" value="/camera_multiplexer/image"/>
    <arg name="depth_topic" value="/camera_multiplexer/depth"/>
  </include>

  <!-- Launch axclient -->
  <node name="axclient"	pkg="actionlib" type="axclient.py"
	args="/motion_detector/find_cabletip"/>
  
  <!-- Launch viewers -->
  <node name="rqt_reconfigure" pkg="rqt_reconfigure" type="rqt_reconfigure"/>
  <node name="$(anon rviz)" pkg="rviz" type="rviz" output="screen"
	args="-d $(dirname)/test.rviz"/>

</launch>
